platform/src/main/java/org/stellar/anchor/platform/observer/stellar/StellarPaymentObserver.java

```diff
@@ -148,8 +148,8 @@ SSEStream<OperationResponse> startSSEStream() {
         new EventListener<>() {
           @Override
           public void onEvent(OperationResponse operationResponse) {
-            if (operationResponse.getTransaction().isPresent()) {
-              metricLatestBlockRead.set(operationResponse.getTransaction().get().getLedger());
+            if (operationResponse.getTransaction() != null) {
```

Comment:
`getTransaction()` returns `Optional`.

```suggestion
            if (operationResponse.getTransaction().isPresent()) {
```

Reply:
Previously, some fields in the response were checked using null, while others were checked using Optional. Now we uniformly check for existence using null.

Reply:
I think this introduces further inconsistency. The `getTransaction()` returns `Optional`. If we want to check the existence using `null`, the `getTransaction()` should return `null` / value instead of `Optional`. 

However, I don't think it is right to make the return value uniform. Whether a function returns `null` or `Optional` should depend on each function's definition. 

I think for functions returning `Optional` should be checked by `isPresent()`. Otherwise,  `!=null`.

Reply:
Hi @lijamie98, I apologize for not mentioning earlier that in the latest version, `getTransaction()` no longer returns `Optional`. You can check the current code here: https://github.com/lightsail-network/java-stellar-sdk/blob/master/src/main/java/org/stellar/sdk/responses/operations/OperationResponse.java#L55

Let me explain the rationale behind removing `Optional`:

- Since we're using `lombok`, there's no need to manually write get functions anymore. Not using `Optional` helps keep things simpler.
- The existence of response fields is heavily dependent on the Horizon version. For instance, a field might exist in Horizon 1.0 but be removed in Horizon 2.0. We want users to have the flexibility to determine field existence based on their specific Horizon version. This way, we avoid tight coupling between the SDK version and Horizon versions.